login/pam.c

```diff
@@ -29,6 +33,11 @@
 
 #define MODULE_NAME "pam_glome"
 
+#ifndef PAM_GLOME_LINUX
+void pam_syslog(void *pamh, ...) { (void)(pamh); }
```

Comment:
Should we implement this in terms of `syslog` instead?

Reply:
With `syslog` it might end up in a different log file or with different format than the other PAM modules so not sure about that.

Reply:
To be fair: I have only made this compile and did not test against PAM's Darwin implementation. I suspect that way lies a lot of pain. My main worry here would be how to do it properly on the other BSD flavors.

For instance pam_krb5 documents these options:

* debug: turns on debugging via [syslog](https://linux.die.net/man/3/syslog)(3). Debugging messages are logged with priority LOG_DEBUG.
* debug_sensitive: turns on debugging of sensitive information via [syslog](https://linux.die.net/man/3/syslog)(3). Debug messages are logged with priority LOG_DEBUG.

Russ Albery wrote https://github.com/rra/pam-krb5/blob/main/portable/pam_vsyslog.c which I find amusing. Presumably we could handle compatibility the same way...

And there I was thinking that PAM is the proper abstraction between different OSes. ;-)